### PR TITLE
Add instructions on how to use in standalone mode

### PR DESCRIPTION
**Pull request by costis**
The library can be used standalone but this isn't mentioned anywhere. This PR adds instructions on how to use it without rails/grape.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -91,14 +92,34 @@ API.logger = Sapience.logger
...
+### Standalone
+Add the gem:
+
+```ruby
+gem "sapience"
+```
+
+Somewhere early in your code execute the following:
+```ruby
+Sapience.configure do |config|
```
Comment (errm) [inline README.md]: I guess we need a require somewhere too?

**Comment by errm** [comment]
There is also a warning logged when started in this configuration, perhaps we should remove this?

**Comment by costis** [comment]
@errm I have another PR that fixes the warning. Let me push it and maybe we merge them?

**Comment by reevoo-samuel** [comment]
Er, hello?

This Pull Request is too damn old! Merge or close this, sucka.

**Comment by reevoo-samuel** [comment]
Er, hello?

This Pull Request is too damn old! Merge or close this, sucka.

**Comment by reevoo-samuel** [comment]
Er, hello?

This Pull Request is too damn old! Merge or close this, sucka.

**Comment by reevoo-samuel** [comment]
Er, hello?

This Pull Request is too damn old! Merge or close this, sucka.

**Comment by reevoo-samuel** [comment]
Er, hello?

This Pull Request is too damn old! Merge or close this, sucka.

**Comment by reevoo-samuel** [comment]
Er, hello?

This Pull Request is too damn old! Merge or close this, sucka.

**Comment by reevoo-samuel** [comment]
Er, hello?

This Pull Request is too damn old! Merge or close this, sucka.

**Comment by reevoo-samuel** [comment]
Er, hello?

This Pull Request is too damn old! Merge or close this, sucka.

**Comment by reevoo-samuel** [comment]
Er, hello?

This Pull Request is too damn old! Merge or close this, sucka.

**Comment by reevoo-samuel** [comment]
Er, hello?

This Pull Request is too damn old! Merge or close this, sucka.

**Comment by reevoo-samuel** [comment]
Er, hello?

This Pull Request is too damn old! Merge or close this, sucka.

**Comment by reevoo-samuel** [comment]
Er, hello?

This Pull Request is too damn old! Merge or close this, sucka.

**Comment by reevoo-samuel** [comment]
Er, hello?

This Pull Request is too damn old! Merge or close this, sucka.

**Comment by reevoo-samuel** [comment]
Er, hello?

This Pull Request is too damn old! Merge or close this, sucka.

**Comment by reevoo-samuel** [comment]
Er, hello?

This Pull Request is too damn old! Merge or close this, sucka.

**Comment by reevoo-samuel** [comment]
Er, hello?

This Pull Request is too damn old! Merge or close this, sucka.

**Comment by reevoo-samuel** [comment]
Er, hello?

This Pull Request is too damn old! Merge or close this, sucka.

**Comment by reevoo-samuel** [comment]
Er, hello?

This Pull Request is too damn old! Merge or close this, sucka.

**Comment by reevoo-samuel** [comment]
Er, hello?

This Pull Request is too damn old! Merge or close this, sucka.

**Comment by reevoo-samuel** [comment]
Er, hello?

This Pull Request is too damn old! Merge or close this, sucka.

**Comment by reevoo-samuel** [comment]
Er, hello?

This Pull Request is too damn old! Merge or close this, sucka.

**Comment by reevoo-samuel** [comment]
Er, hello?

This Pull Request is too damn old! Merge or close this, sucka.

**Comment by reevoo-samuel** [comment]
Er, hello?

This Pull Request is too damn old! Merge or close this, sucka.

**Comment by reevoo-samuel** [comment]
Er, hello?

This Pull Request is too damn old! Merge or close this, sucka.

**Comment by reevoo-samuel** [comment]
Er, hello?

This Pull Request is too damn old! Merge or close this, sucka.

**Pull request merged**